lms/templates/navigation.html

```diff
@@ -51,7 +51,20 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${branding_api.get_logo_url(is_secure)}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+        <%
+        logo_url = None
+        if data:
+            if data.get('third_party_auth_hint', False):
+                logo_url = branding_api.get_enterprise_customer_logo_url(provider_id=data['third_party_auth_hint'])
```

Comment:
we should just access context fields in the view -- let's try to eliminate these direct calls to `branding_api`